starknet-foundry/crates/cast/src/main.rs

```diff
@@ -204,31 +184,27 @@ async fn main() -> Result<()> {
                         new.output_path.clone(),
                         new.overwrite.unwrap_or(false),
                     )?;
-                    starknet_commands::multicall::new::print_new_result(result.as_str());
+                    println!("{result}");
```

Comment:
should this be here?

Reply:
Yes, definitely. The previous function did the same. I think there is no reason to keep such a function

Reply:
Aaah, yes I understand now. Good. :+1: